companies/hekde.json

```diff
@@ -0,0 +1,29 @@
+{
+    "slug": "hekde",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "insurance"
+    ],
+    "name": "HEK Hanseatische Krankenkasse",
+    "address": "Datenschutzbeauftragter\nWandsbeker Zollstraße 86-90\n22041 Hamburg",
```

Comment:
No need to add the `Datenschutzbeauftragter`—[that is added automatically by our website](https://github.com/datenanfragen/data#addresses).